routine-adc.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __ROUTINE_ADC__
+#define __ROUTINE_ADC__
+
+int getADC_Data()
+{
+  return analogRead(SENSOR_PIN);  
+}
+#endif
```

Comment:
can you rename this file to data-adc.h
the same as we already have data-median.h